framework/transport/ieee1905_transport/include/mapf/transport/ieee1905_transport.h

```diff
@@ -64,16 +64,20 @@ class Ieee1905Transport {
     // this table holds all the network interfaces that are to be used by
     // the transport.
     //
-    // interface index (if_index) is used as Key to the table
+    // interface name (ifname) is used as Key to the table
     struct NetworkInterface {
-        int fd =
-            -1; // the file descriptor of the socket bound to this interface (or -1 if inactive)
+        // the file descriptor of the socket bound to this interface (or -1 if inactive)
```

Comment:
If you're anyway changing this, please also doxygenify it:
```suggestion
        /// the file descriptor of the socket bound to this interface (or -1 if inactive)
```

Reply:
I had no idea this should be using `///` instead of `//`. Done.